### ISSUE TITLE
Doesn't handle escaped backslashes

### ISSUE THREAD

**Issue by billpeet** [pheuter/svelte-check-rs#108]
Getting false positives on the following expression:

```svelte
<button onclick={() => open('C:\\Users\\')}>Open</button>
```

Shows:
`Error: unclosed tag: <div> (parse-error)`

Seems to be mishandling the escaped backslashes, it's treating the `'C:\\Users\\'` as if the final quote `'` is escaped.
